app/controllers/annotations_controller.rb

```diff
@@ -154,6 +154,15 @@ def update
         return
       end
     end
-    @annotation_text.update(content: params[:content])
+
+    if params[:annotation_text] && params[:annotation_text][:change_one] && params[:annotation_text][:change_one] == '1'
+      new_text = @annotation_text.dup
```

Comment:
Don't `dup` here, explicitly create the new annotation text inside the transaction with the relevant fields.